### ISSUE TITLE
TickId bigger than Int32.MaxValue

### ISSUE THREAD

**Issue by Nucs** [mathpaquette/IQFeed.CSharpApiClient#2]
Hey,
During accessing history data of `msft-nasdaq`, in parsing of `TickMessage` inside the constructor (see code below),
the following line was received:
`"2018-04-17 17:51:22.000000,96.0700,1061909,0,0.0000,0.0000,4145784264,O,19,143A"`

The parsing code inside constructor:
```C#
var values = msg.Split(',');
return new TickMessage(
    DateTime.Parse(values[0]),
    float.Parse(values[1]),
    int.Parse(values[2]),
    int.Parse(values[3]),
    float.Parse(values[4]),
    float.Parse(values[5]),
    int.Parse(values[6]), //this line is tickId that attempts to parse 4145784264.
    char.Parse(values[7]),
    int.Parse(values[8]),
    values[9]);
```

### Fix:
Change `TickId` to `ulong` (or `long` if there is a possibility to receive -1).

**Comment by mathpaquette** [comment]
Sure let me fix it tonight

**Comment by mathpaquette** [comment]
@Nucs fixed in the 1.3.1 branch.
